lms/envs/devstack.py

```diff
@@ -396,10 +396,10 @@ def should_show_debug_toolbar(request):
 # in it's path. Re-calling derive_settings doesn't work because the settings was already
 # changed from a function to a list, and it can't be derived again.
 
-# from .common import _make_mako_template_dirs
-# ENABLE_COMPREHENSIVE_THEMING = True
-# COMPREHENSIVE_THEME_DIRS = [
-#     "/edx/app/edxapp/edx-platform/themes/"
-# ]
-# TEMPLATES[1]["DIRS"] = _make_mako_template_dirs
-# derive_settings(__name__)
+from .common import _make_mako_template_dirs
+ENABLE_COMPREHENSIVE_THEMING = True
+COMPREHENSIVE_THEME_DIRS = [
+    "/edx/src/adg-edx-theme/edx-platform/"
+]
+TEMPLATES[1]["DIRS"] = _make_mako_template_dirs
+derive_settings(__name__)
```

Comment:
Edx [docker file](https://github.com/edx/configuration/blob/master/docker/build/edxapp/Dockerfile) use lms.jml and studio.yml files for building configuration, maybe in future we can use these files for theming, because lms.yml also have `COMPREHENSIVE_THEME_DIRS`

From docker file
```
COPY docker/build/edxapp/studio.yml /edx/etc/studio.yml
COPY docker/build/edxapp/lms.yml /edx/etc/lms.yml
```

Reply:
@ahmed-faraz46 I have just check /edx/etc/lms.yml and /edx/etc/studio.yml exists in your current lms-shell for AGD, can you try adding a theme path in `lms.yml`

Reply:
@ahmed-faraz46 I have just check `/edx/etc/lms.yml` and `/edx/etc/studio.yml` exists in your current lms-shell for AGD, can you try adding a theme path in `lms.yml`

Reply:
Let's discuss it on a call.

Reply:
As per discussion, we can keep code as is